test/DocumentFormat.OpenXml.Linq.Tests/OpenXmlPartTests.cs

```diff
@@ -36,10 +36,13 @@ public void RootXElement_Get_NewPart_NullAndSynchronized()
             // Arrange.
             using var stream = new MemoryStream();
             using var wordDocument = WordprocessingDocument.Create(stream, WordprocessingDocumentType.Document);
+
+            wordDocument.TrySetPartRootXElementFeature();
```

Comment:
@twsouthwick, will this call be required before the feature can be used? Or can this be avoided?

Reply:
Right now, it's required because it needs to register to events which need to be unhooked and require disposal. The feature to provide disposal right now requires to be done at the package level

Reply:
Unfortunately, having to call this method explicitly would render the API pretty much unusable. For example, when passing a `WordprocessingDocument` to multiple methods that transform the `WordprocessingDocument`, each method would have to ensure the feature is enabled. Countless calls would have to be added to the codebase.

Reply:
Found a workaround. This is no longer needed (it will now verify it has the feature, and if not add it to the parent).